src/libraries/System.Runtime.InteropServices.JavaScript/tests/System.Runtime.InteropServices.JavaScript.UnitTests/System/Runtime/InteropServices/JavaScript/JavaScriptTestHelper.cs

```diff
@@ -27,6 +28,7 @@ public partial class JavaScriptTestHelper
         public static partial string ReboundMemberEcho(string message);
 
         [JSExport]
+        [return: JSMarshalAs<JSType.OneWay>]
```

Comment:
Since this is for test code it's probably harmless, but making console.writeline fire-and-forget this way by pushing it into the work queue means that you could get out-of-order console output

Reply:
I will add comment about that. I was aware of it. 
When this happens across threads it like that anyway.